src/test/resources/pages/instructorStudentRecords.html

```diff
@@ -121,16 +120,13 @@ <h4 class="modal-title">Benny Charlés's Profile - More Info</h4>
                                     </div>
                                     <div id="visibility-options0" class="panel panel-default" style="display: none;">
                                         <div class="panel-heading">Visibility Options</div>
-                                        <table class="table text-center" style="color:#000;">
+                                        <table class="table text-center" style="color: #000; background: #fff;">
```

Comment:
I think we can remove it for now. For now, it will not actually affect the actual background color of the table since the parent element of the table has a white background (due to bootstrap's panel class). The same table appears in the instructor's results page, with the same error, which I can fix in #3512